### PR TITLE
Update of Dutch translations

### PR DESCRIPTION
**Pull request by fridzema**
Couple of finished translations

### CODE REVIEW

**Comment by erciccione** [comment]
Hi and thanks for your contribution. Could you update the title in something like "update of Dutch localization" or something like that?

File: translations/monero-core_nl.ts [hunk 1]
```diff
@@ -520,12 +520,12 @@
     <message>
         <location filename="../LeftPanel.qml" line="508"/>
         <source>Seed &amp; Keys</source>
-        <translation type="unfinished"></translation>
+        <translation>Seed & sleutels</translation>
```
Comment (erciccione) [inline translations/monero-core_nl.ts]: Please keep html entities as they are: `&amp;`

**Comment by ProkhorZ** [comment]
Other Dutch translation: https://github.com/monero-project/monero-gui/pull/1130

@fridzema Thanks for your contributions! But it would help if you coordinated with @erciccione to see what other translators are working on. At the moment, there's much work to be done on monero-site, while I'm working on kovri-docs.

**Comment by erciccione** [comment]
See [my comment on #1130 ](https://github.com/monero-project/monero-gui/pull/1130#issuecomment-367315578)

**Pull request closed without merging**